packages/nextjs/app/builders/0xC13F0f21397E268314ce8d9606838AB5D4eD485c/page.tsx

```diff
@@ -0,0 +1,63 @@
+import Image from "next/image";
+import type { Metadata, NextPage } from "next";
+
+export const metadata: Metadata = {
+  title: "Andrei Zaharov — Batch #20",
+  description: "Builder page for Batch #20",
+};
+const walletAddress = "0xC13F0f21397E268314ce8d9606838AB5D4eD485c";
+
+const ZaharovBuilderPage: NextPage = () => {
+  return (
+    <div className="flex flex-col items-center justify-center gap-6 p-10">
+      {/* Avatar */}
+      <Image
+        src="https://avatars.githubusercontent.com/u/37747146?v=4"
+        alt="avatar"
+        width={128}
+        height={128}
+        className="rounded-full border-4 border-purple-500 shadow-lg"
+      />
+
+      {/* Address */}
+      <h1 className="text-xl font-mono text-purple-400">{walletAddress}</h1>
```

Comment:
There's a component in this project that renders the address. You should try using it.